### DISCUSSION TITLE
KTVKDT1

### DISCUSSION THREAD

**Discussion by AntiPontifex** [iptv-org/community#382] [Show and tell]
### Channel ID

KTVKDT1.us

### Stream URL

https://dai2-playlistserver.aws.syncbak.com/media.m3u8?bitrate=968722&session=eba12acc6b264fdab10177a81efd7022

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://dai2-playlistserver.aws.syncbak.com/media.m3u8?bitrate=968722&session=eba12acc6b264fdab10177a81efd7022 (offline)
<img width="829" alt="image" src="https://user-images.githubusercontent.com/7253922/232255754-8e8480c4-36ce-40cd-878c-06b3ee153c8e.png">
